src/utils/blocks.ts

```diff
@@ -128,3 +130,18 @@ export const getShareLink = (
             return undefined;
     }
 };
+
+export const getCommonQa = (qa?: string, customKeys: Array<string> = []) => {
+    const qaObject: Record<string, string> = {};
+
+    if (qa) {
+        const commonKeys = ['container', 'content', 'wrapper', 'image', 'button'];
```

Comment:
Can we relocate it to constant outside of the function?

Reply:
no problem